pkg/network/setup/netstat.go

```diff
@@ -304,8 +304,12 @@ func isGuestAgentIfaceOriginatedFromOldVirtLauncher(guestAgentInterface api.Inte
 
 func updateVMIIfaceStatusWithGuestAgentData(ifaceStatus *v1.VirtualMachineInstanceNetworkInterface, guestAgentIface api.InterfaceStatus) {
 	ifaceStatus.InterfaceName = guestAgentIface.InterfaceName
-	ifaceStatus.IP = guestAgentIface.Ip
-	ifaceStatus.IPs = guestAgentIface.IPs
+	if guestAgentIface.Ip != "" {
```

Comment:
I'm not clear what you are trying to resolve by this, it will be useful if you describe the scenario and the reasoning for this change in the commit message.